### ISSUE TITLE
Cannot find module '@comunica/actor-rdf-join-nestedloop'

### ISSUE THREAD

**Issue by pheyvaer** [comunica/comunica-actor-init-sparql-hdt#85]
I did a fresh install via 
```
npm i -g @comunica/actor-init-sparql-hdt
``` 
using Node v12.16.0 and when running
```
comunica-sparql-hdt
```
I get the following error:
```
internal/modules/cjs/loader.js:984
  throw err;
  ^

Error: Cannot find module '@comunica/actor-rdf-join-nestedloop'
Require stack:
- /home/pheyvaer/.nvm/versions/node/v12.16.0/lib/node_modules/@comunica/actor-init-sparql-hdt/engine-default.js
- /home/pheyvaer/.nvm/versions/node/v12.16.0/lib/node_modules/@comunica/actor-init-sparql-hdt/bin/query.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:981:15)
    at Function.Module._load (internal/modules/cjs/loader.js:863:27)
    at Module.require (internal/modules/cjs/loader.js:1043:19)
    at require (internal/modules/cjs/helpers.js:77:18)
    at Object.<anonymous> (/home/pheyvaer/.nvm/versions/node/v12.16.0/lib/node_modules/@comunica/actor-init-sparql-hdt/engine-default.js:39:137)
    at Module._compile (internal/modules/cjs/loader.js:1157:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1177:10)
    at Module.load (internal/modules/cjs/loader.js:1001:32)
    at Function.Module._load (internal/modules/cjs/loader.js:900:14)
    at Module.require (internal/modules/cjs/loader.js:1043:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/pheyvaer/.nvm/versions/node/v12.16.0/lib/node_modules/@comunica/actor-init-sparql-hdt/engine-default.js',
    '/home/pheyvaer/.nvm/versions/node/v12.16.0/lib/node_modules/@comunica/actor-init-sparql-hdt/bin/query.js'
  ]
}
```

**Comment by pheyvaer** [comment]
Running `comunica-dynamic-sparql-hdt` does seem to work.

**Comment by rubensworks** [comment]
Hmm, this may require a version update of this package.

**Comment by rubensworks** [comment]
Fixed in 1.5.4.
